app/src/main/java/org/p2p/wallet/home/ui/main/HomeFragment.kt

```diff
@@ -87,12 +97,35 @@ class HomeFragment :
         )
     }
 
+    private val cellAdapter = CommonAnyCellAdapter(
+        tokenDelegate(glideManager) { binding, item ->
+            with(binding) {
+                imageViewHideToken.setOnClickListener {
+                    onHideClicked(item.payload)
+                    binding.root.close(animation = true)
+                }
+                contentView.setOnClickListener { onTokenClicked(item.payload) }
+            }
+        },
+        tokenButtonDelegate() { binding, _ ->
+            with(binding) {
```

Comment:
i don't see any use of with(binding) here really
we can remove it and indentation can be less